src/ace/utils/sprite.c

```diff
@@ -21,7 +19,7 @@ tCopBlock *spriteDisableInCopBlockMode(tCopList *pList, tSpriteMask eSpriteMask)
 	}
 
 	// Set instructions
-	ULONG ulBlank = s_uBlankSprite.ulRaw;
+	ULONG ulBlank = (ULONG)s_pBlankSprite;
```

Comment:
my biggest problem with this change is that if someone just wants to use utils/sprite.h and not sprite manager, then there's no indicator that s_pBlankSprite needs to be set to valid alloc prior to spriteDisableInCopBlockMode. I see 2 solutions:

1. pass the blank sprite buffer through param and remove the global altogether
2. specify in doxycomment that it needs to be initialized first (but many ppl skip docs) and add null check (but var might be set to random value depending on how it's declared).

I'd personally lean towards 1 since it gives user less choice to break things.